lisa/sut_orchestrator/libvirt/schema.py

```diff
@@ -64,6 +64,8 @@ class BaseLibvirtNodeSchema:
     disk_img: str = ""
     # Format of the disk image specified above.
     disk_img_format: str = ""
+    # Pass disk_size_gb to resize the guest image os disk
```

Comment:
Please update the comment. Looks like the property name has changed since.

Reply:
Done